Dockerfile

```diff
@@ -1,21 +1,25 @@
-# FIXME: switch to an alpine image
-FROM ubuntu:20.04
+FROM alpine:3.13 AS builder
 
 LABEL maintainer="AnatoleLucet"
 
-ARG BUILD_APT_DEPS="ninja-build gettext libtool libtool-bin autoconf automake cmake g++ pkg-config unzip git"
-ARG DEBIAN_FRONTEND=noninteractive
+ARG BUILD_DEPS="autoconf automake cmake curl g++ gettext gettext-dev git libtool make ninja openssl pkgconfig unzip"
```

Comment:
Maybe this can be replaced by a [virtual package](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#virtual-packages)?

Reply:
Virtual packages looks like a cool feature 👍  (I've never used them before). 

By using a [multi-stage docker build](https://docs.docker.com/develop/develop-images/multistage-build/) I've skipped trying to clean up any of the build tools. Only the second half of the Dockerfile (/1ab3a79639e8ee62239a661e82e36afd3ec53f8d#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R18-R23) ends up in the final image, and that should be just the compiled nvim ouput

Reply:
Ah! My bad, didn't notice the multi-stage build